src/main/java/decodex/recipes/RecipeManager.java

```diff
@@ -1,24 +1,30 @@
 package decodex.recipes;
 
-import decodex.Decodex;
-import decodex.data.exception.RecipeManagerException;
 import java.util.HashMap;
 import java.util.logging.Logger;
 
+import decodex.Decodex;
+import decodex.data.exception.RecipeException;
+import decodex.data.exception.RecipeManagerException;
+import decodex.modules.Module;
+import decodex.ui.messages.ErrorMessages;
+
 /**
  * RecipeManager manages a list of all recipes that were created/loaded.
  */
 public class RecipeManager {
 
     private Logger logger = Decodex.logger;
 
-    HashMap<String, Recipe> recipeList;
+    private HashMap<String, Recipe> recipeList;
+    private String editedRecipeName;
```

Comment:
Perhaps the variable name can be changed to `recipeUnderEditName`, `selectedForEditRecipeName`, or any other variable name that suggests this recipe is undergoing modification.

Reply:
Maybe just simply `editingRecipeName`. We can discuss this as a team.